.github/workflows/github-main.yaml

```diff
@@ -38,11 +38,6 @@ jobs:
         with:
           arguments: build
 
-      - name: Release
```

Comment:
we no longer run the release plugin on main, because we want snapshot versions from the main build.